### PR TITLE
docs(global): add README.md & CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Aksh-Bansal-dev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Proposing new features
+
+## Standard Commit Messages
+
+This project is using the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) standard. Please follow these steps to ensure your
+commit messages are standardized:
+
+- Commit messages should have this format:
+  `<type>[optional scope]: <description>`
+- Type should be `fix`(if fixed some bug), `feat`(if added new feature), `docs`(changes in documentation) or `test`(added new tests)
```
Comment (Samy-33) [inline CONTRIBUTING.md:16]: Let's add `chore` as well for dependency/configuration change. Later if need arises we can update this list to have more.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Scope should be `ui` or `api` or `global`.
```
Comment (Samy-33) [inline CONTRIBUTING.md:17]: Let's add `ci` here

**Review by Samy-33 — CHANGES_REQUESTED** [review]

**Comment by Aksh-Bansal-dev** [comment]
I have updated the type from the [angular commit guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type).

**Review by Samy-33 — APPROVED** [review]
LGTM.

Just a note: when you're done with your changes, request for the review again, so that the reviewer is notified.

image

**Comment by Aksh-Bansal-dev** [comment]
Okay.

**Pull request merged**